src/index.ts

```diff
@@ -229,7 +229,6 @@ module.exports = class ServerlessPlugin {
             });
           } catch (err) {
             this.serverless.cli.log(`Error occurred when adding source code integration: ${err}`);
-            return;
```

Comment:
Will something like not having git installed cause this error? If so, we may want to suppress this message and only have it for actual errors. If not, disregard!

Reply:
Afaik, if you don't have git installed then `await simpleGit.checkIsRepo()` will throw an error and the deploy will stop altogether? (though this is what would have happened previously too, simpleGit.checkIsRepo() is called outside of the try-catch).

I found [this slightly old resolved issue ](https://github.com/steveukx/git-js/pull/433) opened in the simple-git repo which states any errors (such as git not being installed) will cause the function to not return true/false, but to bubble that error up.